### ISSUE TITLE
PHP 5.4 warning - Plugin: plugin_manager v2.7.1

### ISSUE THREAD

**Issue by GoogleCodeExporter** [roundcubevnz/myroundcube#323]
```
I'm using PHP 5.4 and get a PHP warning in my logs as soon as I click on I 
agree (Fairness is our Mission) while using the Plugin Manager (v2.7.1):

[05-Apr-2012 09:01:29 UTC] PHP Warning:  Illegal string offset 'version' in 
C:\inetpub\wwwroot\mail\plugins\plugin_manager\plugin_manager.php on line 472

I also noticed that the warning shows a wrong time stamp (one hour earlier than 
expected):
- I live in time zone Berlin (UTC +1), suppose it's 11:01:29 now
- UTC time: 10:01:29
- Log shows: 09:01:29 UTC (that's UTC -1 in fact..)


```

Original issue reported on code.google.com by `hijack...@gmail.com` on 5 Apr 2012 at 9:13

**Comment by GoogleCodeExporter** [comment]
```
Try this:

if($scope[$plugin] && $props['version']){

Please use plugin manager -> Update Plugins -> "Issue" Link to open tickets in 
future. Thanks!
```

Original comment by `rol...@liebl.ath.cx` on 5 Apr 2012 at 11:38

**Comment by GoogleCodeExporter** [comment]
```
>> if($scope[$plugin] && $props['version']){
Problem solved. Thanks!

>> Please use plugin manager -> Update Plugins -> 
>> "Issue" Link to open tickets in future. Thanks!
I'll do.


```

Original comment by `hijack...@gmail.com` on 5 Apr 2012 at 11:57

**Comment by GoogleCodeExporter** [comment]
```
Fixed in plugin_manager v2.7.2.
```

Original comment by `rol...@liebl.ath.cx` on 6 Apr 2012 at 4:33
- Changed state: **Fixed**
